docs/getting-started/advanced-guide/schema.md

```diff
@@ -0,0 +1,429 @@
+Gaffer will present the undirected edges Vertices in natural ordering, so a client will also see the above edge presented as `A, B`.
+It will be aggregated with any other edge that has the same source and destination and is also undirected.
+For example an undirected Edge that is added, `A -> B`, will be aggregated with another Edge of `B -> A` if they are undirected.
+
+### Entities
+Entities must have a `vertex` field, which is similar to the `source` and `destination` fields on an edge.
+For example, modelling `London -> Paris` requires an entity definition to represent the city and an edge definition to represent the relationship.
+In that example the entity `vertex` field and the edge `source` and `destination` fields would all be of the same type.
+
+### Optional Element fields
+Edges and Entities can optionally have the following fields:
+
+- `description` - A simple string which should provide some context and explain what the element is.
+- `parents` - An array of parent group names. These must relate to the same sort of element as the child, for example an edge cannot have an entity as a parent. Elements can inherit any information from multiple parent elements. Fields will be merged/overridden, so the hierarchy of parents is important. Any fields that are defined in the child element will also merge or override information taken from the parents.
+- `properties` - Properties are defined by a map of key-value pairs of property names to property types. Property types are described in the Types schema.
+- `groupBy` - By default Gaffer uses the element group and its vertices to group similar elements together when aggregating and summarising elements. This groupBy field allows you to specify extra properties (in addition to the element group and vertices) to use for controlling when similar elements should be grouped together and summarised.
+- `visibilityProperty` - When using visibility properties in your graph, ensure the sensitive elements have a visibility property and then set this visibilityProperty field to that property name. This ensures Gaffer knows to restrict access to sensitive elements.
+- `timestampProperty` - When using the timestamp property in your graph, then set this timestampProperty field to that property name, so Gaffer Stores know to treat that property specially. This property allows Store implementations like Accumulo to optimise the way the timestamp property is persisted. For these stores using it can have a very slight performance improvement due to the lazy loading of properties. Setting the timestampProperty is optional and does not affect the queries available to users. For more information [see the timestamp section of the Accumulo Store Reference](../../reference/stores-guide/accumulo.md#timestamp).
```

Comment:
I feel like this is missing a really simple explanation of what the timestamp property is.